app/scienceapi/events/models.py

```diff
@@ -70,15 +72,26 @@ class Event(models.Model):
         help_text='Slug appended to the Mozilla Science event\'s '
                   'url that represents this event',
     )
+    is_virtual = models.BooleanField(
+        default=False,
+        help_text='Check this box if your event is happening virtually, '
+                  'that is it\'s not tied to a particular location.'
+    )
+
+    ALL_TIMEZONES = [(i, i) for i in pytz.all_timezones]
+    timezone = models.CharField(
+        max_length=40,
+        choices=ALL_TIMEZONES,
+        help_text='Timezone of the place where the event will be held. '
+                  'If the event is virtual select any timezone you like. '
+                  'It is required that you input all datetimes in this '
+                  'timezone.'
+    )
+
     date_created = models.DateTimeField(auto_now_add=True)
     date_updated = models.DateTimeField(auto_now=True)
     starts_at = models.DateTimeField()
```

Comment:
Yes, the front-end conversion for virtual events is unavoidable, but I think the in-person events should have timezone as part of their `starts_at` and `ends_at`, so we can just feed that information in without having to also look at another property. As noted, it stores them in UTC, so a 9AM event in Eastern Standard Time would be stored as `14:00:00-05:00`, right? That totally makes sense, as our front-end MomentJS library is well prepared to handle that.